### PR TITLE
Added copy to clipboard option to code-blocks in `Editor` and `EditorContent`

### LINKED ISSUE #809: Add a copy to clipboard button for the `EditorContent` component.
Part of: #752

### PR DESCRIPTION
**Pull request by gaagul**
- Fixes #809 

**Description**

- Added: copy to clipboard option to code-block `Editor`
- Added: copy to clipboard option to code-block in `EditorContent`

**Checklist**

~~- [ ] I have made corresponding changes to the documentation.~~
~~- [ ] I have updated the types definition of modified exports.~~
- [x] I have verified the functionality in some of the neeto web-apps.
- [x] I have added the necessary label (patch/minor/major - If package publish
      is required).

**Reviewers**

@AbhayVAshokan _a

### CODE REVIEW

File: src/components/Editor/CustomExtensions/CodeBlock/CodeBlockComponent.jsx [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
-const CodeBlockComponent = () => (
+const CodeBlockComponent = ({ node }) => (
   <NodeViewWrapper data-cy="neeto-editor-code-block">
     <pre>
+      <Button
+        className="copy-button"
+        icon={Copy}
+        size="small"
+        style="text"
+        onClick={() => copyToClipboard(node?.content?.content[0]?.text)}
```
Comment (AbhayVAshokan) [inline src/components/Editor/CustomExtensions/CodeBlock/CodeBlockComponent.jsx:16]: Also, add a `Toastr.info`.

File: src/styles/editor/_editor-content.scss [hunk 2]
```diff
@@ -53,6 +54,12 @@
     margin-bottom: 16px;
     border-radius: var(--neeto-ui-rounded-sm);
     padding: 16px;
+
+    .copy-button {
+      position: absolute;
+      top: 2px;
+      right: 2px;
```
Comment (AbhayVAshokan) [inline src/styles/editor/_editor-content.scss:61]:
```suggestion
      top: 4px;
      right: 4px;
```

File: src/components/EditorContent/index.jsx [hunk 3]
```diff
@@ -1,23 +1,49 @@
...
+  useEffect(() => {
+    const preTags = editorContentRef.current?.querySelectorAll(
+      ".neeto-editor-content pre"
+    );
+
+    preTags.forEach(preTag => {
+      const button = document.createElement("div");
+      button.className = "copy-button";
+      const handleButtonClick = () => {
+        copyToClipboard(preTag.textContent);
```
Comment (AbhayVAshokan) [inline src/components/EditorContent/index.jsx:36]: Also check if `Toastr.info` works here. To test this out, you might need to add the `ToastContainer` as well. Else, directly test this out in an host app.

**Review by AbhayVAshokan — APPROVED** [review]
@gaagul _a please make these changes as well: https://vimeo.com/837555318/80a16b7352

File: src/components/Editor/CustomExtensions/CodeBlock/CodeBlockComponent.jsx (hunk 1 above)
Comment (gaagul) [reply src/components/Editor/CustomExtensions/CodeBlock/CodeBlockComponent.jsx:16]: @AbhayVAshokan  https://www.loom.com/share/1c82ef6f28684e51a3178b4e1e500b08?sid=9ad7839c-992c-492a-bab9-ac1f2fd0ae15

**Comment by gaagul** [comment]
> @gaagul _a please make these changes as well: https://vimeo.com/837555318/80a16b735

I will make this change along with the addition of language selector feature for codeBlocks.

File: src/components/EditorContent/index.jsx (hunk 3 above)
Comment (gaagul) [reply src/components/EditorContent/index.jsx:36]: @AbhayVAshokan This will also trigger a Toastr.

**Comment by gaagul** [comment]
@AbhayVAshokan I am merging this PR, All the style changes will taken care of in #812

**Pull request merged**